packages/vaadin-messages/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@vaadin/vaadin-messages",
   "version": "20.0.0-alpha1",
   "description": "vaadin-messages",
-  "main": "vaadin-message.js",
-  "module": "vaadin-message.js",
+  "main": "all-imports.js",
+  "module": "all-imports.js",
```

Comment:
Please also add this (and type definitions) to the `"files"` section below:

```js
  "files": [
    "vaadin-*.d.ts",
    "vaadin-*.js",
    "all-imports.d.ts",
    "all-imports.js",
    "src",
    "theme"
  ],
  ```

Reply:
Done